### PR TITLE
[MODULAR] 20 New Hairstyles!

### PR DESCRIPTION
**Pull request by RimiNosha**
# All credit goes to Crumpaloo, I'm just making this PR for them!

## About The Pull Request
[The whole spriteset!](https://github.com/Skyrat-SS13/Skyrat-tg/pull/18970/checks?check_run_id=10919704159)
image

Adds 20 new hairstyles (duh), they're pretty feminine, and range from medium-long to fairly short, with a pretty good variety!

## How This Contributes To The Skyrat Roleplay Experience

More neato hairstyles that palettes don't make me wanna stab my eyes out!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
The game kept crashing cause master is currently supremely fucked.
  
image
image
image
image

</details>

## Changelog

:cl: Crumpaloo
imageadd: 20 new medium-long to short feminine hairstyles, check near the end of the hairstyles list for more info!
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
post side sprites

**Comment by RimiNosha** [comment]
Image diff bot exists. Posting the result page in the post cause I know you guys'll complain non-stop otherwise.

[Link here too for redundancy.](https://github.com/Skyrat-SS13/Skyrat-tg/pull/18970/checks?check_run_id=10919704159)

**Comment by Nerev4r** [comment]
lgtm wtb TM

**Review by tf-4 — APPROVED** [review]
We need to sort out the fucking hair selection list I hate it I hate it I hate it!!!

**Comment by RimiNosha** [comment]
I want to nuke some of the god ugly hairs that were ported from bay, or wherever has a very different artstyle. There are a couple that really, really, badly stand out.

**Review by GoldenAlpharex — APPROVED** [review]
Code checks out, I'd be down for this. Just going to wait on Halcyon's feedback first.

**Comment by theselfish** [comment]
Any reason you didn't send these upstream?

**Comment by RimiNosha** [comment]
I don't want to, plus them constantly going on about sprite bloat, plus some hairs have (somewhat obscure) references, plus me wanting to actually see these in game at all.

**Comment by theselfish** [comment]
> I don't want to, 

Great attitude to have. What if, say, an upstream dev went "oh, don't want to share this fix with the downstream"?
>plus them constantly going on about sprite bloat, 


There have been two separate TG hair PRs in recent months, your own modularization guide says to try TG first .
image



>plus some hairs have (somewhat obscure) references, 


Have you ever played on a TG server? They have an obscene amount of vague and unknown references.
Plus, they're hairs. You don't need to get the reference to like the hairs.
>plus me wanting to actually see these in game at all.


...What? If they don't get merged (I very much doubt they wouldn't) you can try down here. But they'd certainly not dismiss it outright.

Infact, allow me to ask a TG maintainer. Right now.

@JohnFulpWillard Would you allow these sprites upstream?
Wasn't sure which sprite maintainer to mention, apologies.

**Comment by JohnFulpWillard** [comment]
I’m not a spritetainer but I doubt there’s any problem with sprite bloat over accessories/customization. Your best bet is probably asking in tg’s sprinting channels though. 

I also didn’t catch a single reference looking at the list, take that however you want lol

**Comment by RimiNosha** [comment]
Then you're more than welcome to take them upstream, provided you talk to crumpaloo first. I'm not in the headspace to deal with upstream, plus these *are not my sprites*. This was a PR made on behalf of the spriter.

**Comment by theselfish** [comment]
> I’m not a spritetainer but I doubt there’s any problem with sprite bloat over accessories/customization. Your best bet is probably asking in tg’s sprinting channels though.
> 
> I also didn’t catch a single reference looking at the list, take that however you want lol

I know. I wasn't sure which of your sprite maintainers to mention, so I went with a regular maintainer.

**Comment by theselfish** [comment]
> Then you're more than welcome to take them upstream, provided you talk to crumpaloo first. I'm not in the headspace to deal with upstream, plus these _are not my sprites_. This was a PR made on behalf of the spriter.

If I remember rightly, they said, that like the tattoos Drawsstuff made they were free use.
The only thing doing this downstream causes is annoyance if these are ported up there.
Meaning duplicate hairs at best, and errors at worst.

**Comment by Iamgoofball** [comment]
I'll port these upstream when I get back from Japan, they're good hair

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm]:
```suggestion
	name = "Curly Front"
```
Braided Front is a name used upstream.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm]:
```suggestion
/datum/sprite_accessory/hair/skyrat/curly_front
```
Ditto.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm]:
```suggestion
	name = "Spiky 4"
```
One letter off from an upstream hair.

Comment (theselfish) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm:419]: Ignore the "can't see a hair_spikey in your DMI.". Saw it.

Still very close to an existing hair. Needs a more distinct name.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm]:
```suggestion
/datum/sprite_accessory/hair/skyrat/spiky4
```

**Review by Ebin-Halcyon — APPROVED** [review]
They're fine.

**Comment by GoldenAlpharex** [comment]
> I'll port these upstream when I get back from Japan, they're good hair

Tell me about it when you get back.

**Pull request merged**
